src/views/registration-js/content.php

```diff
@@ -187,18 +193,20 @@ class="tribe-tickets__item__attendee__fields__form <?php echo sanitize_html_clas
 			'components/notice',
 			[
 				'notice_classes'  => $notice_classes,
-				'content' => sprintf(
+				'content'         => sprintf(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 8 space(s) between "'content'" and double arrow, but found 9.

<sub>hash: 6a7d3e8ce89f7bb5953dc66ecf7aad14</sub>

Reply:
Done.